src/applications/vre/28-1900/wizard/app.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default function createChapter31Wizard(store, widgetType) {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+
+  if (root) {
+    import(/* webpackChunkName: "Form686CTA" */
```

Comment:
Can you update the value for `webpackChunkName` to Chapter31Wizard

Reply:
updated